lib/utilities/utility.js

```diff
@@ -72,16 +72,22 @@ function annotateChildPods(kubectl, resourceType, resourceName, allPods) {
 }
 exports.annotateChildPods = annotateChildPods;
 function annotateNamespace(kubectl, namespaceName) {
-    let annotate = true;
     const result = kubectl.getResource('namespace', namespaceName);
-    this.checkForErrors([result]);
-    const annotationsSet = JSON.parse(result.stdout).metadata.annotations;
-    if (!!annotationsSet && !!annotationsSet.runUri && annotationsSet.runUri.indexOf(process.env['GITHUB_REPOSITORY']) == -1) {
-        annotate = false;
-        core.debug(`Skipping 'annotate namespace' as namespace annotated by other workflow`);
+    if (result == null) {
+        return { code: -1, stderr: 'Failed to get resource' };
     }
-    if (annotate) {
-        return kubectl.annotate('namespace', namespaceName, constants_1.workflowAnnotations, true);
+    else if (!!result && !!result.stderr) {
+        return result;
+    }
+    if (!!result && !!result.stdout) {
+        const annotationsSet = JSON.parse(result.stdout).metadata.annotations;
+        if (!!annotationsSet && !!annotationsSet.runUri && annotationsSet.runUri.indexOf(process.env['GITHUB_REPOSITORY']) == -1) {

```

Comment:
Extract the repository value outside; don't keep it inline

Reply:
Done